tests/test_base_env.py

```diff
@@ -51,6 +52,9 @@ def test_base_envs():
     with pytest.raises(ValueError, match=r".*not in.*"):
         env.state = bad_state  # type: ignore
 
+    with pytest.raises(NotImplementedError, match=r"Options not supported.*"):
+        env.reset(options={"option": "value"})
```

Comment:
Why do you take an options keyword argument in the first place? If it's never supported and it's not even part for the gymnasium API:

```def render(self) -> RenderFrame | list[RenderFrame] | None:```

Just let python raise a type error?

Reply:
Apologies, that was my bad. All good here.